app/models/piece.rb

```diff
@@ -124,13 +124,21 @@ def never_moved?
     updated_at == created_at
   end
 
+  def active_player
+    if self.game.move_counter % 2 == 0
```

Comment:
thanks!  I figured there was a way to do it, it just didn't come to me!  I'll make that change.